### ISSUE TITLE
Stop using `extension.getURL` + [unlikely: ] Unable to type in URL field

### ISSUE THREAD

**Issue by wsmwk** [ericpromislow/tabhunter#115]
Fixed by #117

Using tabhunter 3.3.0, Firefox 93.0b9, I'm not able to type in or paste into the URL field.

Nothing in error console when attempting to type.
In error console prior to typing...
 Error: Please use $(ref:runtime.getURL). prefs.js:139
    getURLForLocalFile moz-extension://b2278a9f-a522-a941-981c-bfcffd61aadb/popup/prefs.js:139

**Comment by ericpromislow** [comment]
The URL field has always been readonly -- it's used to display the URL of the selected line.

And I see that error message when I launch Tabhunter on Firefox 92.0. It's complaining about the code I use to find the "factory reset" circle-arrow icon, which still shows up, so you can ignore that.

The warning is legitimate though.

Chrome: https://developer.chrome.com/docs/extensions/reference/extension/#method-getURL
```
Manifest V2 Deprecated 58+

Please use runtime.getURL.
```

Firefox: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/extension/getURL

```
This function is deprecated. Please use runtime.getURL.
```

**Comment by wsmwk** [comment]
Thanks Eric

**Comment by ericpromislow** [comment]
Closed in #117
